mmv1/templates/terraform/examples/scc_folder_big_query_export_config_basic.tf.erb

```diff
@@ -0,0 +1,31 @@
+resource "google_folder" "folder" {
```

Comment:
this example needs to grant the `securitycenter.bigQueryExportsEditor` role on the created folder - potentially with a time_sleep delay afterwards.

Reply:
As per our chat, skipping this test as it is the same as the handwritten one.